js/views/modals/listingDetail/Listing.js

```diff
@@ -168,13 +282,30 @@ export default class extends BaseModal {
         shipsFreeToMe: this.shipsFreeToMe,
         ownListing: this.model.isOwnListing,
         displayCurrency: app.settings.get('localCurrency'),
+        shipsFromCountry: app.settings.get('country'),
```

Comment:
I don't think we have that data, but until we do, I would just hide the field rather than putting in the users Local Country, which IMO makes no sense here.